ipatests/test_integration/test_installation.py

```diff
@@ -391,6 +392,19 @@ def get_pki_tomcatd_pid(host):
             break
     return(pid)
 
+def get_ipa_services_pid(host):
+    ipa_services_name = [
+        "krb5kdc", "kadmin", "named", "httpd", "ipa-custodia",
+        "pki_tomcatd", "ipa-otpd", "ipa-dnskeysyncd", "ntpd"
+    ]
+    pid = []
+    for name in ipa_services_name:
+        service_name = services.knownservices[name].systemd_name
+        cmd = host.run_command(
```

Comment:
Tests typically use ``result`` for the result of a remote command execute, not ``cmd``.